modules/nf-core/preseq/lcextrap/main.nf

```diff
@@ -1,7 +1,7 @@
 process PRESEQ_LCEXTRAP {
     tag "$meta.id"
     label 'process_single'
-    label 'error_retry'
+    label 'error_ignore'
```

Comment:
Can we remove the label and make a custom one?

Retry it once, then if it fails the second time we ignore it?

Reply:
`label 'error_retry_ignore'` ?

Reply:
No just define the error strategy at that point 😂

Reply:
Retrying once and then failing sounds great, from what I recall this fails anytime the sample doesn't have enough depth.